components/BillStatus/BillStatus.js

```diff
@@ -13,8 +13,14 @@ const BillStatus = props => {
 
   return (
     <>
-      <Button variant="primary" className="m-1" onClick={handleShowBillStatus}>
-        Status
+      <Button
+        variant="primary"
+        className="m-1 text-truncate"
+        style={{ width: "18em" }}
+        onClick={handleShowBillStatus}
+      >
+        Status -{" "}
+        {documentHistoryActions[documentHistoryActions.length - 1].Action}
```

Comment:
It's possible that the history is empty, in which case the button should just say "Status" (or maybe "Status - Unknown"?)